apps/token-e2e/cypress.config.js

```diff
@@ -40,8 +41,8 @@ module.exports = defineConfig({
     vegaWalletPublicKeyShort: '02ecea…2f65',
     vegaTokenContractAddress: '0xF41bD86d462D36b997C0bbb4D97a0a3382f205B7',
     vegaTokenAddress: '0x67175Da1D5e966e40D11c4B2519392B2058373de',
-    txTimeout: { timeout: 40000 },
-    epochTimeout: { timeout: 10000 },
+    txTimeout: { timeout: 70000 },
```

Comment:
I don't think increasing timeouts is the right thing to do. We should try to decrease transaction time somehow. In case of errors we'll end up with neverending test jobs.

Reply:
I agree, but i don't think we are going to see a decrease in transaction time anytime soon :(